### DISCUSSION TITLE
Reconsidering access modifiers for certain properties and methods.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3660] [Ideas]
I'm currently attempting to make a module for the editor and I'm currently using EditorObjects.selection and LevelObjects.FindLevelObject through reflection. Using reflection works perfectly fine but is quite inconvenient when having to access properties and methods like these. Long story short, I'm wondering if you could reconsider some of the access modifiers in classes like those to make it easier for programmers to work with the editor.

**Comment by SDGNelson** [comment]
Interesting, didn't know anyone was actively working on that kind of thing. I will keep it in mind for future extensibility.

**Comment by DanielWillett** [comment]
I've tried to do something similar and it becomes a lot of reflection to really do much. Some modding capability for the editor would actually be really neat to see.
